.github/workflows/docs-site.yml

```
@@ -0,0 +1,56 @@
+
+  deploy:
+    needs: build
+    runs-on: ubuntu-latest
+    environment:
+      name: github-pages
+      url: ${{ steps.deployment.outputs.page_url }}
+    steps:
+      - name: Deploy to GitHub Pages
+        id: deployment
+        uses: actions/deploy-pages@v4
+
```

YAMLlint (1.37.1)

[error] too many blank lines (1 > 0)

(empty-lines)